src/components/ShellBox/ShellBox.scss

```
@@ -2,15 +2,21 @@
   font-family: var(--mono);
   display: flex;
   flex-direction: column;
-  background-color: var(--black1);
+  background-color: var(--black9);
   border-radius: 0.4rem;
   box-sizing: border-box;
   padding: 0 0 var(--space-24) var(--space-16);
 
   code {
     overflow-x: auto;
     font-family: inherit;
-    color: var(--brand6);
+    color: var(--pink5);
+    width: calc(100% - 20px);
+    line-height: 30px;
+
+    > span.install__text__command {
+      color: #fff;
```

we could use variables here to be make our code more consistent.
```suggestion
      color: var(--black0);
```

---

Good idea 👍